gf-webapp/src/logic/bookingPage/form/Form.tsx

```diff
@@ -111,8 +117,7 @@ export const Form = (props: RoomProps) => {
                 type="text"
                 className={styles.input}
                 name="roomNumber"
-                placeholder={props.roomNumber ?? undefined}
-                value={bookingFormData.roomNumber}
+                value={props.roomNumber ?? undefined}
                 onChange={handleChangeRoom}
                 required
               />
```

Comment:
A lot of repetitive code, It might be better to use a map().

Reply:
It is inputs with different props. What do you mean to you map here?
Yes it's better to change this code on inputs from ui-library, but right now it impossible because inputs which in storybook don't have all properties which has inputs in form